### ISSUE TITLE
Add example for Databricks

### ISSUE THREAD

**Issue by JCZuurmond** [Stevedow99/dbt-cloud-terraform-template#6]
Hi @Stevedow99 , could you please add an example for the Databricks connection?

**Comment by Stevedow99** [comment]
@JCZuurmond Yeah I have some code I need to push with a Databricks and Redshift example, hope to have it up soon!

**Comment by JCZuurmond** [comment]
Thanks! It would be great if you have an example that implements the [dbt cloud Databricks best practices](https://docs.getdbt.com/guides/dbt-ecosystem/databricks-guides/dbt-unity-catalog-best-practices#isolate-your-bronze-aka-source-data).
